### PR TITLE
Revert pointer events change for OrbitControls and TrackballControls.

### LINKED ISSUE #20191: Clicking on the Ammo sample in the r120 does not launch the iron ball
##### Description of the problem

In the previous Ammo sample, clicking with the mouse launch an iron ball.
However, in the `r120` [sample](https://github.com/mrdoob/three.js/blob/r120/examples/physics_ammo_break.html), the iron ball does not appear when the mouse is clicked.
I tried the combination of the r120 sample and the previous version of [OrbitControls.js](https://github.com/mrdoob/three.js/blob/r119/examples/jsm/controls/OrbitControls.js) and the iron ball does appear, so I think this control is probably related.

[r120 Ammo Convex object breaking example result:](https://rawcdn.githack.com/mrdoob/three.js/r120/examples/physics_ammo_break.html)
image

[r119 Ammo Convex object breaking example result:](https://rawcdn.githack.com/mrdoob/three.js/r119/examples/physics_ammo_break.html)
image

##### Three.js version

- [x] r120

##### Browser

- [x] Chrome
- [x] Firefox

##### OS

- [x] Windows

##### Hardware Requirements (graphics card, VR Device, ...)

MacBook Air (BootCamp) Intel(R) UHD Graphics 617 Direct3D11 vs_5_0 ps_5_0

### LINKED ISSUE #20193: Transform Controls Example is unresponsive to mouse input
The [Transform Controls](https://threejs.org/examples/?q=controls#misc_controls_transform) example isn't responding to any mouse input when you try to move the box around. 

This is working however on mobile Safari and mobile Chrome.
##### Three.js version / on the live site
- [ X] r120

##### Browser

- [x] All of them
- [ ] Chrome
- [ ] Firefox
- [ ] Internet Explorer

##### OS
- [X ] macOS

### LINKED ISSUE #20205: Decal example is not working
The original decal example is not working anymore on r120: https://threejs.org/examples/#webgl_decals

Nothing happened when clicking on the object, no error reported in console.

### PR DESCRIPTION
**Pull request by Mugen87**
Fixed #20191.
Fixed #20193.
Fixed #20205.

### CODE REVIEW

**Comment by arodic** [comment]
I just want to note that this pointerEvents upgrade also broke all controls when using stylus (`event.pointerType === 'pen'`)

```javascript
switch ( event.pointerType ) {
   case 'mouse':
       onMouseMove( event );
       break;
}
```

I completely and overwhelmingly support transition to pointerEvents. It is an amazing API that will solve most of the unnecessary conditionals when dealing with different input devices and it will make the controls much simpler. It enables developers to write code that works with any pointer device, including the future ones.  

In my experience, `switch ( event.pointerType )` is unnecessary for most use cases - it defeats the purpose of using pointerEvents. You can treat pointer events as input-device-agnostic API. The only special case in my experience is pointer "hover" that cannot be achieved on traditional touch device. If you need a special hover behavior, this too can be achieved without dealing with pointerType.

In other words, switching to pointerEvents is great, but it should be done without dealing with `pointerType` unless really necessary - which is almost never.

**Comment by Mugen87** [comment]
@mrdoob Since you want to keep the Pointer Events change, I'll close this PR.

**Comment by WestLangley** [comment]
`webgl_multiple_scenes_comparison.html` slider is broken.

I think a hot patch to r.120 is appropriate.

**Comment by WestLangley** [comment]
It does not appear the hot patch fixed the slider example...

**Comment by Mugen87** [comment]
Still working on a solution: #20223

**Comment by WestLangley** [comment]
Oh, I see. You didn't revert the recent changes to the controls. That would have made it a trivial hot patch...

**Comment by mrdoob** [comment]
> `webgl_multiple_scenes_comparison.html` slider is broken.

Should be fixed now.
